composer.json

```diff
@@ -9,6 +9,8 @@
     "pimple/pimple": "^3.0",
     "symfony/event-dispatcher": "~2.7",
     "behat/behat": "^3.0",
+    "phpspec/phpspec": "~2.0",
```

Comment:
Wanted to do that, but then saw behat in non-dev and decided to put phpspec alongside it. Move behat to `dev` as well?